website/docs/docs/core/pip-install.md

```diff
@@ -154,3 +154,107 @@ python -m pip install \
 ```
 
 Or, better yet, just install the package(s) you need!
+
+<VersionBlock firstVersion="1.8">
+
+### Install prereleases of dbt-adapters
+
+A prerelease adapter is a version released before the final, stable version. It allows users to test new features, provide feedback, and get early access to upcoming functionality.
```

Comment:
nice intro! i think the first couple of paragraphs can be consolidated as they're saying basically saying the same thing

Reply:
So something like: 

> A prerelease adapter is a version released before the final, stable version. It allows users to test new features, provide feedback, and get early access to upcoming functionality &mdash; ensuring your system will be ready for the final release.